faq.md

```diff
@@ -0,0 +1,31 @@
+## FAQ questions
+
+### Why use a new ABI? Why not an attribute?
+- Unlike ABIs, there is no syntax for assigning attributes to function
+  pointers. The ABI must be part of the type system so that callers of function
+  pointers know whether or not the function may unwind.
+### What should I do if I want to have C++ functions that unwind into Rust? [DRAFT; TODO]
+- mark declarations in Rust `extern "C unwind"`
+- Do *not* use `catch_unwind` in calling code
+- compile w/ `panic = unwind`
+- Check compiler, linker, & platform documentation (TODO: expand on this)
+- Re-test for every compiler update
+- (TODO: other caveats?)
+### How does cross-language unwinding differ from cross-language
+  `setjmp`/`longjmp`?
+- `setjmp`/`longjmp` across Rust frames is currently guaranteed not to have
+  undefined behavior or to `abort`
+- It should never be assumed that `drop` will be called for objects in
+  intermediate frames traversed by a `longjmp`, but this may occur on certain
+  platforms. Rust provides no guarantee either way. Conversely, on platforms
+  where unwinding is supported, it will always call `drop`.
```

Comment:
What does this last sentence mean ? E.g. `x86_64-unknown-linux-gnu` is a platform where unwinding is supported, but longjmp never drops values living in stack frames. On other platforms like MSVC, do `longjmp`s always drop values on stack frames?

Reply:
Sorry, "it" here should have been "unwinding". You're right, this wasn't clear.